support/src/main/java/org/jenkinsci/plugins/workflow/support/DefaultStepContext.java

```diff
@@ -87,7 +88,7 @@
                 ConsoleLogFilter filter = get(ConsoleLogFilter.class);
                 OutputStream os = new FileOutputStream(la.getLogFile(), true);
                 if (filter != null) {
-                    os = filter.decorateLogger(null, os);
+                    os = filter.decorateLogger((AbstractBuild) null, os);
```

Comment:
No.

This part of the patch is just part of the `post-625` branch, not this PR specifically.